src/main/java/net/minecraftforge/common/BiomeDictionary.java

```diff
@@ -328,6 +329,12 @@ private static void registerVanillaBiomes()
         addTypes(Biomes.SAVANNA_PLATEAU, HOT, SAVANNA, PLAINS, SPARSE, RARE, OVERWORLD, PLATEAU);
         addTypes(Biomes.BADLANDS, MESA, SANDY, DRY, OVERWORLD);
         addTypes(Biomes.WOODED_BADLANDS, MESA, SANDY, DRY, SPARSE, OVERWORLD, PLATEAU);
+        addTypes(Biomes.MEADOW, PLAINS, PLATEAU, OVERWORLD);
+        addTypes(Biomes.GROVE, COLD, CONIFEROUS, FOREST, SNOWY, MOUNTAIN, OVERWORLD);
+        addTypes(Biomes.SNOWY_SLOPES, COLD, SPARSE, SNOWY, MOUNTAIN, OVERWORLD);
+        addTypes(Biomes.JAGGED_PEAKS, COLD, SPARSE, SNOWY, MOUNTAIN, OVERWORLD);
+        addTypes(Biomes.FROZEN_PEAKS, COLD, SPARSE, SNOWY, HILLS, OVERWORLD);
```

Comment:
Isn't this part of the mountain category?

Reply:
It generates lower than the other mountain biomes, wasn´t sure if Hills or Mountain would fit best. Will change it to Mountain :)